client/ayon_core/tools/loader/control.py

```diff
@@ -425,3 +432,28 @@ def _create_event_system(self):
 
     def _emit_event(self, topic, data=None):
         self._event_system.emit(topic, data or {}, "controller")
+
+    def get_current_context_product_types_filter(self):
```

Comment:
This method should expect project name. When you have selected library project, the product types are not the same.